work3/templates/index.html

```diff
@@ -0,0 +1,18 @@
+{% extends "base.html" %}
+
+{% block form1 %}
+<form method="post">
+    {{ form.csrf_token }}
+    {% for field in form if  field.name != 'csrf_token' %}
+        {{ field.label }}<br>
+        {{ field }}<br>
+        {% if field.errors %}
+            <ul>
+                вводить нужно правильно<br>
+                {{ field.errors }}
+            </ul>
+        {% endif %}
```

Comment:
Nne можно сделать чуть иначе 
```suggestion
            <ul>
            {% for error in obj.errors %}
                <li>
                    <span style="color: red;">{{ error }}</span>
                </li>
            {% endfor %}
            </ul>
```